### DISCUSSION TITLE
Feature request: customizable footer area

### DISCUSSION THREAD

**Discussion by craigreyenga** [requarks/wiki#7403] [General]
Many websites have a large footer area, often in a darker colour with the typical set of links: about, contact, privacy policy, and so forth.

I managed to add this to a fork of wiki JS pretty easily. The footer override field in admin was modified to be a text area instead of a text field, for larger multi line input. Rendering was changed to render it as markdown, with auto-link disabled, and raw HTML enabled. This, along with a couple small CSS tweaks allowed creation of a large footer area with various links.

Other than not knowing the size limit for that field in the database, it is working well. So at this point, it would really just be a product management decision on whether to add this to wiki JS in a future release.

**Comment by NGPixel** [comment]
Post on https://feedback.js.wiki/wiki instead.
